### PR TITLE
doc: Fix some typos

### PR DESCRIPTION
**Pull request by eltociear**
accomodate -> accommodate

### CODE REVIEW

**Comment by jonatack** [comment]
If you're going to fix a typo, can you please fix all of them as seen in the spelling linter script `test/lint/lint-spelling.sh`, e.g. I believe:

src/net_processing.h:68: anounce ==> announce
src/netaddress.h:486: compatiblity ==> compatibility
src/test/validation_tests.cpp:78: excercise ==> exercise
test/functional/feature_nulldummy.py:63: unnecssary ==> unnecessary

and update `test/lint/lint-spelling.ignore-words.txt` to ignore all of the current false positives.

**Comment by michaelfolkson** [comment]
Additional typos [suggested](https://github.com/bitcoin/bitcoin/pull/20762#issuecomment-750889701) by @jonatack have now been edited in second commit.

1) Can you [squash](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits) the two commits @eltociear?

2) Are you going to do the second part of @jonatack suggestion @eltociear or do you want someone else to do that in another PR?

> update test/lint/lint-spelling.ignore-words.txt to ignore all of the current false positives.

**Comment by practicalswift** [comment]
ACK a95f3c364b71e122e773ba40ece7b614c79daa9f modulo addressing the nits suggested above

**Comment by theStack** [comment]
LGTM, happy to ACK as soon as the commits are squashed.

**Comment by eltociear** [comment]
Update commit and PR title.

@michaelfolkson I would like to ask another person for the second part.

**Comment by theStack** [comment]
Running the spelling linter script, I saw that there is another instance of "crypted" that could be tackled in this PR:
```
$ ./test/lint/lint-spelling.sh
test/functional/wallet_encryption.py:81: crypted ==> encrypted
```
Will open a follow-up PR that updates the false positives list as soon as this is merged.

**Comment by michaelfolkson** [comment]
False positives are now dealt with in the follow up PR (thanks @theStack). All current and future generations of Atacks are eternally grateful.

Can you then just add the one remaining [typo](https://github.com/bitcoin/bitcoin/pull/20762#issuecomment-752911473) suggestion @eltociear, squash the commits and then this should be ready.

The label of the PR needs to be changed from Build System to Docs by a maintainer.

**Comment by practicalswift** [comment]
ACK 7ce93c752d19f570952e26649134bcbd2447b3a5

Welcome as a contributor @eltociear :)

**Comment by jonatack** [comment]
In this commit, can you please fix the remaining error signaled by the linter after #20817:
```
test/functional/wallet_encryption.py:81: crypted ==> encrypted
```
and prefix the commit with `doc:` rather than `refactor:`

**Comment by eltociear** [comment]
@jonatack  @michaelfolkson @practicalswift @theStack 
A Happy New Year🎍

I'd added fix commit and squash commits.
Thank you for your continuous support.

**Comment by practicalswift** [comment]
re-ACK 62a537652e2ae325980a683fb294ce017f21317f

**Comment by yahiheb** [comment]
ACK 62a537652e2ae325980a683fb294ce017f21317f

**Review by theStack — APPROVED** [review]
ACK 62a537652e2ae325980a683fb294ce017f21317f 🍾

**Review by hebasto — COMMENTED** [review]
Concept ACK.

IIUC, this PR initially was motivated by a change
> accomodate -> accommodate

in the `depends/packages/qt.mk` file.

Currently, the `depends/` directory is ignored by the `lint-spelling.sh` linter.

With the following patch:
```diff
--- a/test/lint/lint-spelling.sh
+++ b/test/lint/lint-spelling.sh
@@ -15,6 +15,6 @@ if ! command -v codespell > /dev/null; then
 fi
 
 IGNORE_WORDS_FILE=test/lint/lint-spelling.ignore-words.txt
-if ! codespell --check-filenames --disable-colors --quiet-level=7 --ignore-words=${IGNORE_WORDS_FILE} $(git ls-files -- ":(exclude)build-aux/m4/" ":(exclude)contrib/seeds/*.txt" ":(exclude)depends/" ":(exclude)doc/release-notes/" ":(exclude)src/leveldb/" ":(exclude)src/crc32c/" ":(exclude)src/qt/locale/" ":(exclude)src/qt/*.qrc" ":(exclude)src/secp256k1/" ":(exclude)src/univalue/"); then
+if ! codespell --check-filenames --disable-colors --quiet-level=7 --ignore-words=${IGNORE_WORDS_FILE} $(git ls-files -- ":(exclude)build-aux/m4/" ":(exclude)contrib/seeds/*.txt" ":(exclude)depends/config.guess" ":(exclude)depends/config.sub" ":(exclude)doc/release-notes/" ":(exclude)src/leveldb/" ":(exclude)src/crc32c/" ":(exclude)src/qt/locale/" ":(exclude)src/qt/*.qrc" ":(exclude)src/secp256k1/" ":(exclude)src/univalue/"); then
     echo "^ Warning: codespell identified likely spelling errors. Any false positives? Add them to the list of ignored words in ${IGNORE_WORDS_FILE}"
 fi
```

more typos have been revealed:
```
depends/packages/qt.mk:220: accomodate ==> accommodate
depends/patches/fontconfig/gperf_header_regen.patch:5: inadvertant ==> inadvertent
depends/patches/native_cctools/ld64_disable_threading.patch:11: noticable ==> noticeable
depends/patches/qt/freetype_back_compat.patch:12: compatibile ==> compatible
depends/patches/qt/no-xlib.patch:56: glpyh ==> glyph
```

**Comment by michaelfolkson** [comment]
Happy New Year @eltociear! Let us know if you're happy to add these additional @hebasto typos or if you'd prefer someone to do these in a follow up PR.

**Comment by hebasto** [comment]
[Another suggestion](https://github.com/bitcoin/bitcoin/pull/20817#discussion_r550874370) is to combine one more typo: s/Keypair/Key pair/ in https://github.com/bitcoin/bitcoin/blob/ae8f79713543d3db24a886cc27340b1570c2657d/src/bench/verify_script.cpp#L27

**Comment by eltociear** [comment]
@michaelfolkson @hebasto Thank you!
Added 5 typo fix.

File: depends/patches/qt/no-xlib.patch
Comment (fanquake) [inline depends/patches/qt/no-xlib.patch:56]: You can't modify the actual contents of a patch like this, because it will no longer apply. See all the CIs that are now failing:
```bash
patching file qtbase/src/plugins/platforms/xcb/qxcbcursor.cpp
Hunk #5 FAILED at 595.
1 out of 6 hunks FAILED -- saving rejects to file qtbase/src/plugins/platforms/xcb/qxcbcursor.cpp.rej
funcs.mk:267: recipe for target '/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.9.8-ed5ca6e91a4/.stamp_preprocessed' failed
make: *** [/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.9.8-ed5ca6e91a4/.stamp_preprocessed] Error 1
make: Leaving directory '/tmp/cirrus-ci-build/depends'
```

**Comment by fanquake** [comment]
Thanks for your contribution. However to move things along I'm going to fix this up and consolidate it with some other changes. Your commit will still be attributed to you.

**Pull request closed without merging**
